components/fxa-client/src/oauth.rs

```diff
@@ -17,6 +17,12 @@ use url::Url;
 const OAUTH_MIN_TIME_LEFT: u64 = 60;
 
 impl FirefoxAccount {
+    /// Fetch a temporary access token using the saved refresh token.
```

Comment:
nit: I would say "short-lived" rather than "temporary", I'm not sure "temporary" gives quite the right impression.